app/services/code_climate/api/repository.rb

```diff
@@ -0,0 +1,23 @@
+module CodeClimate
+  module Api
+    class Repository < CodeClimate::Api::Object
+      def summary
+        default_branch_most_recent_snapshot&.summary
+      end
+
+      def id
```

Comment:
I would change the name of the method with something like, `id_retriever` or `repository_id`

Reply:
Will change it. Thanks!